README.md

```diff
@@ -144,6 +144,15 @@ clients can make of use of the general functionality contained in
 
 ### Development
 
+enrich-classpath is important for development of Java-related features in Orchard.
+
+You can fire up a repl (and nrepl server) that uses cider-nrepl and enrich-classpath like so:
+
+```bash
+# or `make lein-repl`
+make
```

Comment:
Does this make sense to be a default target? (I recall those usually compiled or packaged stuff) Perhaps `make lein-repl` is enough.